README.md

```diff
@@ -2,7 +2,7 @@
 
 # Portifolio project
 
-> One paragraph statement about the project.
+> This is a trial project
 
 ![screenshot](./app_screenshot.png)
 
```

Comment:
Update your README file to more descriptive changes.

Reply:
- [ ] [REQUIRED] Nice Job giving reasons for the requested change, however I kindly suggest you phrase your comments as suggestions and be polite

:x: Wrong
Update your README file to more descriptive changes.

:heavy_check_mark: Right
I kindly suggest that it will be better to update your README file to be more descriptive.

Reply:
- Kindly edit your review as suggested by the previous reviewer to `I kindly suggest that it will be better to update your README file to be more descriptive.`